config/initializers/0_clowder-config.rb

```diff
@@ -10,7 +10,10 @@ def build_endpoint_url(endpoint, config)
 
   config = ClowderCommonRuby::Config.load
 
-  ENV['SSL_CERT_FILE'] = config.tlsCAPath if config.tlsCAPath
+  if config.tlsCAPath
+    system('cat /etc/pki/tls/certs/ca-bundle.crt /cdapp/certs/service-ca.crt > /tmp/combined.crt')
```

Comment:
[CWE-377 Insecure Temporary File](https://cwe.mitre.org/data/definitions/377.html)

Reply:
@vkrizan This will be running in a container.  CWE-377 should not apply.

Reply:
After this is merged, someone please create a cleanup of this thing to pure ruby.